src/main/java/net/mcreator/ui/modgui/ProcedureGUI.java

```diff
@@ -131,6 +131,17 @@ private void regenerateProcedure() {
 			hasResultTriggerLabel.setIcon(null);
 			sideTriggerLabel.setIcon(null);
 
+			// Check that no local variable has the same name as one of the dependencies
+			for (var dependency : dependenciesArrayList) {

```

Comment:
Why are you using `var` instead of `Dependency`?

Reply:
The variable name makes it clear that it is a `Dependency`, and anyway this wouldn't be the first usage of `var`